$OpenBSD: patch-src_compiler_x86_parms_lisp,v 1.1 2019/03/21 22:06:54 sthen Exp $

Move the static spaces to an address which should be available under
both ld.bfd and ld.lld

Index: src/compiler/x86/parms.lisp
--- src/compiler/x86/parms.lisp.orig
+++ src/compiler/x86/parms.lisp
@@ -204,7 +204,7 @@
 #!+sunos     (!gencgc-space-setup #x20000000 :dynamic-space-start #x48000000)
 #!+freebsd   (!gencgc-space-setup #x01000000 :dynamic-space-start #x58000000)
 #!+dragonfly (!gencgc-space-setup #x01000000 :dynamic-space-start #x58000000)
-#!+openbsd   (!gencgc-space-setup #x3d000000 :dynamic-space-start #x8d000000)
+#!+openbsd   (!gencgc-space-setup #x11000000 :dynamic-space-start #x8d000000)
 #!+netbsd    (!gencgc-space-setup #x20000000 :dynamic-space-start #x60000000)
 #!+darwin    (!gencgc-space-setup #x04000000 :dynamic-space-start #x10000000)
 
